packages/asc-ui/src/components/ContextMenu/ContextMenu.stories.tsx

```diff
@@ -7,7 +7,7 @@ import { ReactComponent as Linkedin } from '@datapunt/asc-assets/lib/Icons/Linke
 import { ReactComponent as Email } from '@datapunt/asc-assets/lib/Icons/Email.svg'
 import { ReactComponent as Print } from '@datapunt/asc-assets/lib/Icons/Print.svg'
 import { AscCore } from '../../styles'
-import { ContextMenu, ContextMenuItem, Icon } from '../..'
+import { ContextMenu, ContextMenuItem, IconStyle } from '../..'
```

Comment:
Only use components from asc-ui, as they will be available to import from the @datapunt/asc-ui npm package